docs/jsonapi-spec.md

```diff
@@ -536,14 +536,40 @@ Creates a new collection in the current namespace.
 ```json
 {
   "createCollection": {
-    "name": "purchase"
+    "name": "default_collection",
```

Comment:
We don't have concept of default_collection should we call some_collection or something?

Reply:
+1 to changing it since we do/will have "default_namespace" so there might be assumption this is similarly default name if not specified, or has some other special significance.

Reply:
good point - will use some_collection as a Spec example

Reply:
done in most recent commit